CONTRIBUTING.md

```diff
@@ -72,17 +73,94 @@ Once merged to the main branch, `po` files and any documentation change will be
 
 ### Required dependencies
 
-TODO
+This project has several build dependencies. You can install these dependencies from the top of the source tree using the `apt` command as follows:
+
+```shell
+sudo apt update
+sudo apt build-dep .
+```
 
 ### Building and running the binaries
 
-TODO
+The project consists of the following binaries:
+
+* `authd`: The main authentication service.
+* `pam_authd.so` and `pam_authd_exec.so`: A PAM module and its helper library.
```

Comment:
The components are more:

```suggestion
* `pam_authd.so`: A PAM native module (used by GDM)
* `pam_authd_exec.so`, `authd-pam`: A PAM module and its helper executable (used by other PAM applications).
```

Note that this duality would be avoided by using #121.

Reply:
I wouldn't talk about the helper here. Too much details.